### DISCUSSION TITLE
Does Etsy Limit The Number of Search Result Records You Can Access?

### DISCUSSION THREAD

**Discussion by showtroylove** [etsy/open-api#1188] [Q&A] [answered]
Steps to reproduce: Perform a query using the API findAllListingsActive:

{{etsy_v3_url}}/listings/active?keywords=Free Mug&limit=1&sort_on=score&taxonomy_id=1062&offset=12001

You may receive the following error message despite there being:

```
{
    "error": "Offset provided is greater than the maximum offset allowed."
}

{
    "count": 32166,
    "results": [...]
}
```
You might expect this error if you tried an offset beyond the upper range of the result set namely, 32167.  Another unpublished detail?

**Comment by Gareth064** [comment]
Yes it has a ceiling for how many results returned, there was another discussion somewhere regarding this. Can't find it though so it might have been an "issue" but all the issues got deleted or hidden when everything was changed to using "discussions"

**Reply by showtroylove** [reply]
Apologies, let me clarify. It appears that you cannot access a dataset beyond the 12,000th row. For example, if I call it and limit the result to 1 record (i.e., limit = 1), this gives me the count and, depending on sorting, the 1st record in that dataset or query result. 

So, while I can get at least 1 record, if I attempt to retrieve records from 12,001 to 12,050 (for example), which is only 50 records, it seems to deny access to the result rows beyond 12,000. This is something an API should mention.  

To be clear, I have not exceed or even come close to my rate-limits.

**Reply by Gareth064** [reply, marked as answer]
Cool. This is what I was looking for

https://github.com/etsy/open-api/discussions/1030#discussioncomment-6177474

or Direct info from the docs that i mention in that comment

https://developers.etsy.com/documentation/essentials/urlsyntax/#pagination

**Reply by showtroylove** [reply]
That's does clearly state the 12, 000 number.  I read this before, but didn't understand it to mean you cannot access records beyond 12,000.  If this is the case why would there data layer allow results sets larger than 12,000.  In essence, a huge swath of query results are unreachable data.

**Reply by showtroylove** [reply]
I would not like my listing to fall into this huge dead zone of Etsy listings.

**Reply by Gareth064** [reply]
What app feature would be interested in getting every possible listing?
